EVM and High Level Languages.md

```diff
@@ -0,0 +1,95 @@
+# The EVM and High-Level Smart Contract Programming Languages
+
+## Introduction
+
+Bitcoin has always been the highest volume and value cryptocurrency in digital money. It runs on blockchain technology. Blockchain is a peer to peer, public, encrypted, trust-less ledger. Script is the name of the scripting language that constructs Bitcoin. Script is a mid to lower capability programming language. Public and private key transacting is the model for signing transactions. Besides transacting digital assets, there is no extra utility built into Bitcoin. 
+
+Ethereum Blockchain technology offers limitless use case scenarios. A virtual computer deployed on the Ethereum Blockchain creates many possible use cases. Smart contracts are programs written in a high-level programming language. Think of JavaScript syntax with a name of Solidity or Vyper.  These languages build Smart contract code that compiles for use on the Ethereum Blockchain. This computing system revolutionizes privacy, security, and freedom otherwise unrealized in present-day governance.
+
+## Ethereum Virtual Machine Feature Overview
+
+High-level is a vague description of a computing platform's capability. Precise performance metrics would go beyond the scope of this article. The Ethereum Blockchain is a powerful and isolated virtual computing environment. Computing features of the Ethereum Virtual Machine (EVM) include: 
+
```

Comment:
Could you add a tiny bit of detail on what these mean, or at least maybe link to the wiki or something like that?